packages/react/src/enhancers/withLoadImportMap.tsx

```diff
@@ -23,8 +23,14 @@ export function withLoadImportMap<T extends WithLoadImportMapProps>(
 ) {
   const WithLoadImportMap = (props: T): JSX.Element => {
     const loadImportMapContext = useContext(ImportMapReactContext);
-
-    return <Component {...props} loadImportMap={props.loadImportMap || loadImportMapContext} />;
+    const loadClientImportMap = props.loadImportMap || loadImportMapContext;
+    return (
+      <Component
+        {...props}
+        loadClientImportMap={loadClientImportMap}
+        loadImportMap={loadClientImportMap}
+      />
+    );
```

Comment:
We don't need to apply any of those changes, right?
We still need to pass only `loadImportMap` and we still need fallback to `props.loadImportMap`